e2e/sanity_test.go

```diff
@@ -62,5 +65,13 @@ func testSanity() {
 		_, _, err = kubectl("exec", "-n", "topolvm-system", "daemonset/topolvm-node", "--", "test", "-e", path)
 		return sanity.PathIsOther, err
 	}
+
+	if os.Getenv("TEST_THIN_DEVICECLASS") == "true" {
+		// csi.storage.k8s.io/fstype=xfs,topolvm.cybozu.com/device-class=sbvglvm
```

Comment:
What is `sbvglvm`? (The device set is actually set to `thin` at line 73.)

Reply:
I'll fix this.

Reply:
Fixed.